src/js/components/Calendar/Calendar.js

```diff
@@ -708,7 +708,7 @@ const Calendar = forwardRef(
             sizeProp={size}
             fillContainer={fill}
           >
-            <StyledDay otherMonth sizeProp={size} fillContainer={fill}>
```

Comment:
why was this taken out?

Reply:
Previously the days of the week were receiving the `otherMonth` styling, but we don't want this to happen anymore and instead they should get the same styling as the regular days of the month.

Reply:
okay thank you for clarifying